safe/gui/widgets/dock.py

```diff
@@ -841,7 +848,11 @@ def layer_changed(self, layer):
 
         # Do nothing if there is no active layer - see #1861
         if not self._has_active_layer():
-            send_static_message(self, getting_started_message())
+            if is_plugin_installed('EmergencyMapper'):
```

Comment:
Perhaps you can also put this if statement as a method. So that you don't need to change the plugin name later if you want to update it. And make a list of the coflicting plugins (if there is more than one).

But, the current one is working great also. My above comment is not mandatory

Reply:
Yes, it what I thought, I wanted to make something clean, but it's very very low priority for me. I doubt we will have another plugin again. And maybe in a few days or weeks, they will fix their plugin so we will have to remove these functions.